test/tools/router_check/coverage.cc

```diff
@@ -53,7 +53,55 @@ double Coverage::report() {
   return 100 * static_cast<double>(covered_routes_.size()) / num_routes;
 }
 
+void Coverage::printMissingTests(std::set<uint64_t> const& all_route_names,
+                                 std::set<uint64_t> const& covered_route_names) {
+  std::set<uint64_t> missing_route_names;
+  std::set_difference(all_route_names.begin(), all_route_names.end(), covered_route_names.begin(),
+                      covered_route_names.end(),
+                      std::inserter(missing_route_names, missing_route_names.end()));
+  std::set<uint64_t>::iterator it = missing_route_names.begin();
+  for (const auto& host : route_config_.virtual_hosts()) {
+    for (const auto& route : host.routes()) {
+      if (it != missing_route_names.end() && std::stoull(route.name()) == *it) {
+        std::string route_match_type = "";
+        std::string route_match_string = "";
+        switch (route.match().path_specifier_case()) {
+        case envoy::api::v2::route::RouteMatch::kPrefix: {
```

Comment:
is there a way to dump the entire proto here instead of many if elses? This will quickly become stale with changes in the proto